package.json

```diff
@@ -59,13 +77,17 @@
     "rollup-plugin-scss": "^4.0.0",
     "rollup-plugin-tsconfig-paths": "^1.5.2",
     "sass": "^1.77.4",
+    "storybook": "^8.0.5",
     "ts-jest": "^29.0.3",
     "ts-node": "^10.9.2",
     "tsconfig-paths": "^4.2.0",
-    "typescript": "^4.9.5"
+    "typescript": "^4.9.5",
+    "vite": "^4.5.3",
+    "vite-tsconfig-paths": "^4.3.2"
   },
   "peerDependencies": {
     "big.js": "^6.2.2",
+     "@storybook/react": "^8.0.5",
```

Comment:
should this be a peer dep?

Reply:
if i set it to devDependencies compiler  complains that it needs to be moved to dependencies. shall i move it to dependencies instead ?

Reply:
i moved it to devdependencies now and compiler doesnt complain. strange i couldnt make that error go away until i moved them to peer and now moving them back to dev it works. will update

Reply:
no if i move them to devDependencies am still getting :  
'@storybook/react' should be listed in the project's dependencies, not devDependencies.eslint[import/no-extraneous-dependencies]